.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.words": ["FUSCHIA"]
```

Comment:
yeah, this is from one of my extensions, tbh there is a setting to set this at user level rather than project level, so I must have hit the wrong button there. I will get rid of this though